TACC/Core-Styles
# hotfix: tup 644 footer not sticky during edge case

## Overview

Sticky footer even if content is in between main and footer.

## Related

- [TUP-644](https://jira.tacc.utexas.edu/browse/TUP-644)

## Changes

- **added** content to reveal edge case
- **changed** selector to cover edge case

## Testing

1. Open http://localhost:3000/components/detail/sticky-footer.
2. Verify even with unexpected content, footer still sticks to bottom of page.

## UI

| Before, _no Edge Case_ | Before, _w/ Edge Case_ | _After_, w/ Edge Case | _Mimic Fix_ on TUP |
| - | - | - | - |
| ![BEFORE no edge case for bug](https://github.com/TACC/Core-Styles/assets/62723358/49b2c339-e16e-45f1-8ba3-b81be6bb5a7a) | ![DURING show edge case for bug](https://github.com/TACC/Core-Styles/assets/62723358/b338f3df-17e9-4e40-94af-dd64175c8c1e) | ![AFTER fix edge case for bug](https://github.com/TACC/Core-Styles/assets/62723358/00f80199-f163-43e1-b0f7-6bc25d3ce590) | ![AFTER mimicking fix on TUP](https://github.com/TACC/Core-Styles/assets/62723358/ed1346e4-de17-428b-a269-c5b8159dd99c) |
